### DISCUSSION TITLE
Changing and supplementing information regarding the work of developers with the API

### DISCUSSION THREAD

**Discussion by mensakh** [nocodb/nocodb#5359] [Features]
A week ago, I decided to start using NocoDB because I liked the simplicity of Airtable, the second thing that caught my attention was the API, which had the same functions as Airtable.

First I found some API examples on the page - [link to page](https://apis.nocodb.com)

After that, I started trying to find even more features and eventually came across a page - [link to page](https://all-apis.nocodb.com)

One of the main problems of each of these pages is that not everywhere are written parameters that should be passed to send requests, the second problem is lack of examples with filled data for the correct sending of the request. Often I have to try everything to understand how I need to use any API method with the correct parameters and their transfer. So, for example, I don't know in which particular request NocoDB will process parameters, data or json.

**Comment by o1lab** [comment]
We've revamped API docs and you will see an update next week. If you would like to see changes run `npm run redoc` within packages/nocodb folder.

**Reply by mensakh** [reply]
Okay, but right now I can't use the npm command, as I'm completely new to nodejs and all sorts of commands from the terminal, please upload the html code to any file hosting service if possible and please send the link.
